api/migrations/0009_profile.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.7 on 2019-12-01 22:28
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('api', '0008_auto_20191110_0322'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Profile',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('points', models.IntegerField(default=0)),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+    ]
```

Comment:
Just curiosity, do we have to update these migration py files according to changes? For example, I also updated 0001_initial.py file for `list_input_size` on my local ...but didn't push to team's master branch...

Reply:
Yes - it's better to check the migration files in. It's probably not absolutely necessary, but it could lead to some sync problems if you don't.